Dockerfile

```diff
@@ -25,44 +25,42 @@
 # Use Python-2.7:
 FROM python:2.7-slim
 
-COPY scripts/provision-web.sh /tmp/
-
-# Install CERN Open Data Portal web node pre-requisites:
-RUN /tmp/provision-web.sh
+# Configure CERN Open Data Portal instance:
+ENV INVENIO_WEB_INSTANCE=cernopendata
+ENV INVENIO_INSTANCE_PATH=/usr/local/var/cernopendata-instance
 
 # Add CERN Open Data Portal sources to `code` and work there:
 WORKDIR /code
 ADD . /code
 
-# Run container as user `invenio` with UID `1000`, which should match
-# current host user in most situations:
-RUN adduser --uid 1000 --disabled-password --gecos '' invenio && \
-    chown -R invenio:invenio /code
-USER invenio
```

Comment:
User 1000 can be removed, but then check volume mounting on Docker. E.g. files created within the container won't have the same user identity as the laptop user, which is usually a problem.  (but only in DEV setups, not in PROD setups, so for the sake of speeding up the OpenShift deployment, this could be done...)